libs/s25main/BasePlayerInfo.h

```diff
@@ -37,4 +38,8 @@ struct BasePlayerInfo
     /// Returns index of color in PLAYER_COLORS array or -1 if not found
     int GetColorIdx() const;
     static int GetColorIdx(unsigned color);
+
+    // 0: Initial
+    // 1: Added portraitIndex
+    static inline constexpr int getCurrentVersion() { return 1; }
```

Comment:
Can you move this to the cpp file? No need to have this in the header causing recompiles on change

Reply:
Done!